pkg/connector/invitation.go

```diff
@@ -69,6 +71,9 @@ func (i *invitationResourceType) List(ctx context.Context, parentID *v2.Resource
 		PerPage: pt.Size,
 	})
 	if err != nil {
+		if isNotFoundError(resp) {
```

Comment:
Because this is the resources sync and we don't check `IsWarning` for resources I don't think this will prevent the error. I think maybe log and return `return nil, "", nil, nil`